.github/workflows/merge.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - develop
       - master
+  workflow_dispatch:
```

Comment:
allows this to be run wihtout a pr (mostly for me sanity checking or re reunning things or if we want to cut a release without a pr)

Reply:
I had a look at the file separately as it looked odd - l shouldn't the `workflow_dispatch` key have a value, or some child keys indented below it? At first glance I assumed env: was a child of this but now seeing it without this review block in the way I don't quite understand how this is valid.

Reply:
So we don't really pass any inputs into this workflow, so i don't think we need anything.  Truthfully im not entirely sure we need this to begin with, i was adding it as a way of running CI without needing a PR up. so if you just need to sanity check things or something odd fails and you want to re run the branch